app/src/main/java/com/project/pv239/customtimealarm/fragments/SetAlarmFragment.java

```diff
@@ -1,36 +1,256 @@
+                final TimePickerDialog.OnTimeSetListener listener = new TimePickerDialog.OnTimeSetListener() {
+                    @Override
+                    public void onTimeSet(TimePicker view, int hourOfDay, int minute) {
+                        mAlarm.setHour(hourOfDay);
+                        mAlarm.setMinute(minute);
+                    }
+                };
+                new TimePickerDialog(getContext(), listener, mAlarm.getHour(), mAlarm.getMinute(), true).show();
+            }
+        });
+        final ArrayAdapter<CharSequence> travelAdapter = ArrayAdapter.createFromResource(getContext(),
+                R.array.travel_mode, android.R.layout.simple_spinner_item);
+        travelAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        mTravelMode.setAdapter(travelAdapter);
+        mTravelMode.setSelection(mAlarm.getTravelMode());
+        mTravelMode.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                mAlarm.setTravelMode(position);
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> parent) {
+
+            }
+        });
+        final ArrayAdapter<CharSequence> trafficAdapter = ArrayAdapter.createFromResource(getContext(),
+                R.array.traffic_model, android.R.layout.simple_spinner_item);
+        trafficAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        mTrafficModel.setAdapter(trafficAdapter);
+        mTrafficModel.setSelection(mAlarm.getTrafficModel());
+        mTrafficModel.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                mAlarm.setTrafficModel(position);
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> parent) {
+
+            }
+        });
+        mMorningSet.setProgress(mAlarm.getMorningRoutine());
+        mMorningSet.setMax(240);
+        mMorningView.setText(String.valueOf(mAlarm.getMorningRoutine()));
+        mMorningSet.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
+            @Override
+            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
+                mAlarm.setMorningRoutine(progress);
+                mMorningView.setText(String.valueOf(mAlarm.getMorningRoutine()));
+            }
+
+            @Override
+            public void onStartTrackingTouch(SeekBar seekBar) {
+            }
+
+            @Override
+            public void onStopTrackingTouch(SeekBar seekBar) {
+            }
+        });
         Log.d("FRAGMENT CREATION", "i am fragment for alarm id " + mAlarm.getId());
         return view;
     }
```

Comment:
pre kristove rany :D :D :D :D

Reply:
jojo ja viem java :D ale ako vravis budeme to rozdelovat len sa mi to uz nechcelo